app/controllers/admin/settings/base_controller.rb

```diff
@@ -22,11 +22,11 @@ def create
         else
           respond_to do |format|
             format.html do
-              redirect_to admin_config_path, alert: "😭 #{result.errors.to_sentence}"
+              redirect_to admin_config_path, alert: "❗️❗️ #{result.errors.to_sentence}"
             end
 
             format.json do
-              render json: { error: "😭 #{result.errors.to_sentence}" }, status: :unauthorized
+              render json: { error: "❗️❗️ #{result.errors.to_sentence}" }, status: :unauthorized
```

Comment:
I wonder how these symbols are going to be read by the screen reader. I understand they're added to catch the attention of the user since we don't have a way to differentiate between errors and non errors in Snackbar and we should have a global notification area for errors or something and that's outside the scope wrt the conversation with Pawel. 

I think the Admin team should figure this one out in a future RFC/PR

cc. @aitchiss @ludwiczakpawel

Reply:
We probably don't need any emojis in the error message.

Reply:
I would love it if instead of emojis in the text we could have an optional svg/img icon in the snackbar to convey an error message vs a success/info message. You're right @rhymes  in that screen reader behaviour with emojis can vary widely, and what it announces might not be something we intend (e.g. VO on Chrome announces 😭  as "loudly crying face", but this can be different when using a different screen reader/browser/etc)

Reply:
[Bonus context on how to mark-up emoji accessibly](https://dev.to/s_aitchison/accessible-images-icons-and-emojis-53g7#emojis) 😄

Reply:
I think I agree with Nick here - removing the emojis from the error messaging. They were already in place when I started this task; I changed the crying-face emoji to the exclamation points as I thought that was more indicative of an error. However, using the emojis re not worth the a11y drawbacks.

Eventually, I would love our snackbars to be customizable in terms of color perhaps, to indicate a successful transaction, as opposed to a failed one. But that's another PR.